python_modules/dagster-graphql/dagster_graphql/schema/asset_graph.py

```diff
@@ -171,16 +172,22 @@ def resolve_jobs(self, _graphene_info):
         ]
 
     def resolve_partitionKeys(self, _graphene_info):
```

Comment:
one question i have about this once it goes through gRPC is how frequently it's going to be called and where it will be called. calls into gRPC can be pretty slow (particularly in cloud where it's asynchronous since it goes through the agent - think, like, a second or more per call potentially) - so if we were calling this once per asset and there were a bunch of assets in a single page load, that sounds like it could be prohibitively expensive. But I don't really have context on the feature this is being built for and how many times partitionKeys will be called in a page load

Reply:
(but if this is on a page for a single asset, then that could be fine - the partition set page works similarly, since it needs to make a gRPC call on page load to load the set of partitions, and that sometimes requires access to user code)